content/erguotou/drone-instant-access.md

```diff
@@ -0,0 +1,52 @@
+---
+date: 2020-03-07T00:00:00+08:00
+title: InstantAccess(即时达)
+author: erguotou
+tags: [ notifications, InstantAccess, 即时达 ]
+logo: instant_access.svg
+repo: erguotou520/drone-instant-access
+image: erguotou/drone-instant-access
+---
+
+The InstantAccess plugin posts build status messages to your account. The below pipeline configuration demonstrates simple usage:
+
+```yaml
+---
```

Comment:
For all the other examples we are not using a full pipeline, just the steps required for the plugin. Please use the other plugins as an example.

Reply:
So just remove
```
---
kind: pipeline
name: default
```
?

Reply:
Doc updated

Reply:
Exactly like you have done it now.